spack-ext/repos/spack-stack/packages/neptune-env/package.py

```diff
@@ -47,5 +47,6 @@ class NeptuneEnv(BundlePackage):
     # Basic Python dependencies that are always needed
     depends_on("py-f90nml", type="run")
     depends_on("py-python-dateutil", type="run")
+    depends_on("py-pyyaml", type="run")
```

Comment:
ESMX needs this, thus we need to build this regardless of whether we build the full Python stack for NEPTUNE or not.